src/System.Memory/tests/ReadOnlyBuffer/ReadOnlySequenceTests.Common.cs

```diff
@@ -103,6 +103,44 @@ public void CheckEndReachableDoesNotCrossPastEnd()
             Assert.Equal(200, seq.Length);
         }
 
+        [Fact]
+        public void CanGetFirst()
```

Comment:
We have other First tests right?

Reply:
Yes; though not mismatched multi segment buffer sizes (which this is)